src/components/file-browser/FilesTable.ts

```diff
@@ -91,21 +92,31 @@ const FilesTable = (props: FilesTableProps) => {
               headerRenderer: () => h(HeaderCell, ['Name']),
               cellRenderer: ({ rowIndex }) => {
                 const file = files[rowIndex];
```

Comment:
A lot of whitespace changes here. Reviewing with whitespace ignored (https://github.com/DataBiosphere/terra-ui/pull/4635/files?w=1) makes this diff smaller.